src/GitHub.VisualStudio/AssemblyResolverPackage.cs

```diff
@@ -0,0 +1,88 @@
+﻿using System;
+using System.IO;
+using System.Reflection;
+using System.Diagnostics;
+using System.Globalization;
+using Microsoft.VisualStudio;
+using Microsoft.VisualStudio.Shell;
+using System.Linq;
+
+namespace GitHub.VisualStudio
+{
+    // This is the Git service GUID, which fires early and is used by GitHubService.
+    [ProvideAutoLoad(Guids.GitSccProviderId)]
+
+    // This fires before ShellInitialized and SolutionExists.
+    [ProvideAutoLoad(VSConstants.UICONTEXT.NoSolution_string)]
+    public class AssemblyResolverPackage : Package
+    {
+        // list of assemblies that should be resolved by name only
+        static readonly string[] ourAssemblies =
+        {
+            "GitHub.UI",
+            "GitHub.VisualStudio.UI",
+            "System.Windows.Interactivity"
+        };
```

Comment:
If I understand the code correctly, then we're loading things from this list if they're referenced from xaml, which means they're referenced by partial name. If that's the case, then this list should probably also include `GitHub.UI.Reactive`, since that's also referenced from xaml. Perhaps audit all our xaml files to make sure this list includes all our referenced?